runner/kubernetes.go

```diff
@@ -253,3 +312,60 @@ func NewKubernetesJobClient(name string, namespace string) (*KubernetesJobClient
 	}
 	return &KubernetesJobClient{Clientset: clientset, JobName: name, Namespace: namespace}, nil
 }
+
+//for local implementations
```

Comment:
remove comments

Reply:
I'll add a variable that changes which client initializer is called depending on the implementation rather than have a comment. good call.